tools/generate-helpers/pg-table-bindings/main.go

```diff
@@ -139,11 +138,12 @@ func main() {
 			"OptionsPath":       path.Join(packagenames.Rox, props.OptionsPath),
 			"JoinTable":         props.JoinTable,
 			"PermissionChecker": props.PermissionChecker,
```

Comment:
The `.PermissionChecker` itself is used in order to call the object for permission checks in `store.go.tpl`. It is also used where I think we should now use `.Obj.HasPermissionChecker` to activate the permission checker code generation (e.g. the pattern was changed line 541, but not line 605).